doc/specs/vulkan/man/VkMemoryPropertyFlags.txt

```diff
@@ -22,14 +22,14 @@ ename:VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT::
     Identifies a memory pool that can be mapped into host memory address
     space and thus is accessible by the host.
 
-ename:VK_MEMORY_PROPERTY_HOST_NON_COHERENT_BIT::
+ename:VK_MEMORY_PROPERTY_HOST_COHERENT_BIT::
```

Comment:
Apparently we flipped the sense of those bits at some point and then didn't reflect it in the spec. Good catch, and it's obvious how to fix the text to match the name change, but this is an actual spec change so will have to go back to the working group.